lib/axiosHttpClient.ts

```diff
@@ -181,8 +181,8 @@ export class AxiosHttpClient implements HttpClient {
 
       try {
         if (operationResponse.bodyAsText) {
-          const contentType = operationResponse.headers.get("Content-Type");
-          if (contentType === "application/xml" || contentType === "text/xml") {
+          const contentType = operationResponse.headers.get("Content-Type") || '';
+          if (contentType.includes("application/xml") || contentType.includes("text/xml")) {
```

Comment:
`includes()` is a dangerous thing to use here because it will match `application/xmlblah`, which may not be desired. I really recommend splitting this and doing an equality check instead.

Reply:
Please have a look at 4b1ddc5